how-to-peercc-callstats.md

```diff
@@ -0,0 +1,51 @@
+# PeerCC Sample - Integration with callstats.io
+
+[callstats.io](https://www.callstats.io) is a service which uses WebRTC statistics to monitor an analyze voice and video communications to deliver better user experiences.  The [PeerCC](https://www.github.com/webrtc-uwp/PeerCC) repo contains sample code to illustrate how to use the WebRTC UWP SDK with the callstats.io service.  To enable this functionality follow the instructions below.
+
+## Requirements
+
+* [OpenSSL](https://slproweb.com/products/Win32OpenSSL.html)
+* Tested with version 1.1.1a 
+
+> NOTE: The WebRTC SDK development environment requires the installation of Strawberry Perl and adding it to the system path.  It may be necessary to add the OpenSSL 1.1.1a installation location to the path to prevent issues.  To check the location of the openssl binary, type 'where openssl.exe' in the command prompt.
+
+## Callstats REST API authentication steps
+
+* Set `App ID`: copy from your application settings to Config.cs

```

Comment:
Best ask @ivanab to review this to be still true.

Reply:
I checked this, documentation is correct.